tools/data-handler/src/utils/ClingoParser.ts

```diff
@@ -0,0 +1,261 @@
+/**
+    Cyberismo
+    Copyright © Cyberismo Ltd and contributors 2024
+
+    This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License version 3 as published by the Free Software Foundation.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public
+    License along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+interface Result {
+  key: string;
+  labels: string[];
+  links: {
+    key: string;
+    linkType: string;
+    displayName: string;
+    linkDescription?: string;
+  }[];
+  policyChecks: {
+    successes: { testSuite: string; testCase: string }[];
+    failures: { testSuite: string; testCase: string; errorMessage: string }[];
+  };
+  deniedOperations: {
+    transition: { transitionName: string; errorMessage: string }[];
+    move: { errorMessage: string }[];
+    delete: { errorMessage: string }[];
+    editField: { fieldName: string; errorMessage: string }[];
+    editContent: { errorMessage: string }[];
+  };
+  results: Result[]; // Nested results
+  [key: string]: any;
```

Comment:
Instead of "any", what would be the potential actual types? 
string|number|boolean|...

Reply:
I improved the types. For now, all fieldtypes are parsed as strings in the parser